src/components/Restoration/RestorationType.vue

```diff
@@ -133,3 +140,18 @@ export default class RestorationType extends Vue {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.valid-title {
+  color: $gray9;
+}
+.invalid {
+  color: $app-red;
+}
+
+.valid-text {
+  color: $gray7;
+}
```

Comment:
Instead of adding spans with a class and then selecting that class here, I highly recommend you select the existing Vuetify classes (same as my comment in other PR).